### PR TITLE
Release Update

### PR DESCRIPTION
**Pull request by isamauny**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,108 +75,172 @@ By default, entries in the OpenAPI Explorer pane are sorted alphabetically. If y
...
-![Details for specific issues](https://github.com/42Crunch/vscode-openapi/blob/master/images/Details%20for%20specific%20issues.gif?raw=true)
+![](https://img.shields.io/badge/Learning%20More!-Blue) Watch this [video](https://42crunch.com/free-user-faq/?utm_campaign=IDE&utm_medium=referral&utm_source=vscode#Free-API-Security-Testing-FAQs-4) to learn more about 42Crunch Scan.
 
-3. For a quick overall look, check the counts in the Status Bar. The different icons match the severity of the issue:
+APIs which thoroughly enforce compliance to an established contract are far more resilient to all types of attacks.
 
-- Error: critical or high
-- Warning: medium
-- Info: low
+![](https://img.shields.io/badge/Important-red)  You should only use 42Crunch Scan against APIs that you own, but not those of third parties.
```
Comment (lruusuvuori) [inline README.md:203]: "You must only use 42Crunch Scan against APIs that you own, not those of third parties." to give this more weight and less hesitancy (the "but" doesn't fit in well).

File: README.md [hunk 2]
```diff
@@ -88,108 +75,172 @@ By default, entries in the OpenAPI Explorer pane are sorted alphabetically. If y
...
+- **A credential** : most likely, your API is using some form of authentication, like an API Key or token. You need a valid credential to provide to the scan engine. 
 
-#### Individual fixes
+- **The URL** when the API is deployed.  
 
-1. Perform the Security Audit by clicking the **42c** button.
-2. When you click some of the error lines in your OpenAPI file, you will see the QuickFix icon show up on the left of the line.
-3. Click the icon and select the QuickFix to apply.
-4. Tweak the values in the inserted code snippet as you see fit.
+  ![](https://img.shields.io/badge/Warning-orange)  We recommend that you do <u>not</u> target a production system. While the tool does not try to inject malicious payloads, it is possible that the API implementation is not resilient enough to handle the tests and may crash or behave unexpectedly.
```
Comment (lruusuvuori) [inline README.md:215]: I think we've elsewhere been much more strict and said that they MUST not target production. I think this may even be in our Ts&Cs. Of course, this is now in the context of freemium where they have not yet had to sign up to our Ts&Cs, so perhaps our approach could be more lenient then? In  any case, something to think about in terms of the wording here.

File: README.md [hunk 3]
```diff
@@ -88,108 +75,172 @@ By default, entries in the OpenAPI Explorer pane are sorted alphabetically. If y
...
 
-![Quick Fixes in VSCode](/images/Quick_Fixes_in_VSCode.gif)
+When you first launch a scan, you will be presented with the scan configuration viewer. The scan configuration is generated automatically from the OpenAPI file you chose to scan.
```
Comment (lruusuvuori) [inline README.md:217]: "When you first launch a scan, you will be presented with the scan configuration viewer."

This is very hard to read, but I don't know enough about the context to choose how to best fix it. Consider one of these as the alternative (in my order of preference):
- "When you launch a scan, you see the scan configuration viewer."
- "When you first launch a scan, you see the scan configuration viewer."
- "When you first launch a scan, you are presented with the scan configuration viewer."

File: README.md [hunk 4]
```diff
@@ -88,108 +75,172 @@ By default, entries in the OpenAPI Explorer pane are sorted alphabetically. If y
...
-2. When you click some of the error lines in your OpenAPI file, you will see the QuickFix icon show up on the left of the line.
-3. Click the icon and select the QuickFix to apply.
-4. Tweak the values in the inserted code snippet as you see fit.
+  ![](https://img.shields.io/badge/Warning-orange)  We recommend that you do <u>not</u> target a production system. While the tool does not try to inject malicious payloads, it is possible that the API implementation is not resilient enough to handle the tests and may crash or behave unexpectedly.
 
-![Quick Fixes in VSCode](/images/Quick_Fixes_in_VSCode.gif)
+When you first launch a scan, you will be presented with the scan configuration viewer. The scan configuration is generated automatically from the OpenAPI file you chose to scan. 
 
-#### Bulk fixes
+- For each operation in the OpenAPI file, a request is created. You can test individual requests using the **Try** button top-right.
```
Comment (lruusuvuori) [inline README.md:219]: I think the button is called "Try It"?

File: README.md [hunk 5]
```diff
@@ -88,108 +75,172 @@ By default, entries in the OpenAPI Explorer pane are sorted alphabetically. If y
...
 
-If your OpenAPI file contains multiple similar issues - for example, a parameter with the same name is missing the pattern definition - you can fix all such occurrences in one go:
+  ![](./images/ScantryIt.png)
 
-1. Fix the first occurrence and supply the proper values in the code snippet.
-2. Go to the next occurrence and use the bulk QuickFix. This bulk QuickFix will reuse the values that you used previously for the parameter with the same name.
+- Requests can be arranged into testing scenarios: a scenario combines one or multiple requests, for example you need to create a resource before you can view it. The editor allows you to extract data as variables from a request execution to inject it into the next step.
+	
+	Similarly to requests, you can test an individual scenario to ensure it is built correctly, before starting a scan: the scan engine will execute the scenario and if successful, launch automatically dozens of test, using the data provided in the OpenAPI file. It will send bad verbs, bad data types, bad data formats as well as authentication tests.
```
Comment (lruusuvuori) [inline README.md:225]: Lose the commas after "correctly" and "dozens of tests" (missing the plural s there, btw), they make the sentence too choppy and hard to follow.

"It will send..." would be better to be more explicit what you refer to here, as there are so many candidates what the "it" could be in this one; use "The scan will send..." instead.

File: README.md [hunk 6]
```diff
@@ -88,108 +75,172 @@ By default, entries in the OpenAPI Explorer pane are sorted alphabetically. If y
...
+	
+	![](/Volumes/MYDATA/Product/vscode-openapi/images/ScanScenarios.png)
+	
 
-![Bulk Fixes in VSCode](/images/VSCode_OpenAPI_bulk_security_fixes.gif)
+![](https://img.shields.io/badge/Learning%20More!-green)  <link to video tutorial> 
 
 ## Miscellaneous commands
 
 - You can load results of Security Audit from a file for a currently open OpenAPI file by running command `42Crunch: Load Security Audit report from file` from the command palette
```
Comment (lruusuvuori) [inline README.md:234]: Hmmm, not sure if this should be "Security Audit" or "42Crunch Audit". See which you think works better here, both could be justified (but for different reasons).

File: README.md [hunk 7]
```diff
@@ -88,108 +75,172 @@ By default, entries in the OpenAPI Explorer pane are sorted alphabetically. If y
...
 
-## Integration with 42Crunch Platform
+## Network Requirements
```
Comment (lruusuvuori) [inline README.md:236]: Network requirements

File: README.md [hunk 8]
```diff
@@ -88,108 +75,172 @@ By default, entries in the OpenAPI Explorer pane are sorted alphabetically. If y
...
 
-This extension features optional integration with [42Crunch Platform](https://platform.42crunch.com/). To enable it please configure your credentials by running `42Crunch: Update platform credentials` command from the command palette.
+In order to use the 42Crunch Freemium services, we need access to the following URL: https://stateless.42crunch.com. You may need to ask your administrators to add a firewall rule to allow the connection.
```
Comment (lruusuvuori) [inline README.md:238]: Simplify and align: "To use the 42Crunch freemium services,..."

**Review by ak1394 — APPROVED** [review]
